actors/sunbird-lms-mw/actors/common/src/main/java/org/sunbird/learner/actors/tac/UserTnCActor.java

```diff
@@ -133,6 +178,20 @@ private void acceptTNC(Request request) {
     }
   }
 
+  private Map<String, Object> createDBFormatAllTncAccepted(Map<String, Object> allTncAcceptedMap) {
```

Comment:
Can you name the function as intent? Like flatten() or so.
Its difficult to look at this otherwise. At least a comment is needed for sure.

Reply:
`convertTncMapObjectToJsonString`  Storing tnc map object to json string in cassandra